test/ex_machina_test.exs

```diff
@@ -69,6 +69,30 @@ defmodule ExMachinaTest do
     }
   end
 
+  test "build_pair/2 builds 2 factories" do
+    records = MyApp.Factories.build_pair(:user, admin: true)
+
+    expected_record = %{
+      id: 3,
+      name: "John Doe",
+      admin: true
+    }
+
```

Comment:
I think this newline can be removed since it feels like it's part of the assertion phase. Same for the test below. What do you think?